gradio/data_classes.py

```diff
@@ -25,6 +25,8 @@
     from pydantic import BaseModel as BaseModelV1
     from pydantic import ValidationError, schema_of
 
+    JsonValue = Any
```

Comment:
We can be more specific no? Something like
```suggestion
    JsonValue = Union[Dict, List]
```